Zappy_IA_Report/DTC/Video/Tutorial4.py

```
@@ -0,0 +1,32 @@
+import pandas as pd
+
+df = pd.read_csv('500hits.csv', encoding='latin-1')
+print(df.head())
+
+df = df.drop(columns= ['PLAYER', 'CS'])
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)